### DISCUSSION TITLE
洛雪为什么有的歌可以下载，有的不行

### DISCUSSION THREAD

**Discussion by UR1411** [lyswhut/lx-music-desktop#2022] [Q&A]
![image](https://github.com/user-attachments/assets/376fba78-ba58-4eb9-98a9-5f92c42f6d36)
如图，灰色的歌不能下载，其他的可以

**Comment by UR1411** [comment]
解决了，原来是音源的问题，有的音源有下载功能，有的没有

**Reply by Bifaxin** [reply]
你好能不能提供下好的源？

**Reply by UR1411** [reply]
https://wwp.lanzouq.com/b02fq9w8d
密码：lxmusic

**Reply by yuzhufeng** [reply]
也不行了。。
